.bazelci/presubmit.yml

```diff
@@ -166,17 +166,6 @@ tasks:
     - "-//tests/core/cgo:generated_versioned_dylib_client"
     - "-//tests/core/cgo:generated_versioned_dylib_test"
     - "-//tests/core/cross:proto_test"
-    - "-//tests/core/go_embed_data:go_default_test"
```

Comment:
Is the implication that these were not getting tested at all before?

Reply:
Some of the test may be broken but not caught. In fact `//tests/extras/go_embed_data:go_default_test` is still broken, so I am not able to enable it in this PR.

Reply:
Some of the test may be broken but not caught. In fact `//tests/extras/go_embed_data:go_default_test` is still broken, so I am not able to enable it in this PR.